src/utils/http_client.rs

```diff
@@ -1,4 +1,4 @@
-use crate::config::default_request_timeout;
+use crate::config::{default_request_timeout, internal_client_connect_timeout};
```

Comment:
Since `internal_client_connect_timeout` is not used in the tests we can include it only on non-test builds (avoids the warning regarding an unused dependency in tests). Example:

```rust
use crate::config::default_request_timeout;
#[cfg(not(test))]
use crate::config::internal_client_connect_timeout;
```

Reply:
Yeah I was also bothered by the warnings where there was seemingly no way around it. Good suggestion!